website/docs/core-concepts/projects/configuration/stores.mdx

```diff
@@ -18,6 +18,7 @@ Currently, the following stores are supported:
 - [AWS SSM Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html)
 - [Artifactory](https://jfrog.com/artifactory/)
 - [Redis](https://redis.io/)
+- [Google Secret Manager](https://cloud.google.com/secret-manager)
```

Comment:
NIT: could you alphabetize the list of supported stores (and add the new GSM docs section in the same order)